sycl/test/basic_tests/range_offset_fit_in_int.cpp

```diff
@@ -0,0 +1,178 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -D__SYCL_ID_QUERIES_FIT_IN_INT__=1 %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
Your change seems to be device-agnostic. Do we really need to run it on all devices?

Reply:
Removed excess runs.